README.md

```diff
@@ -44,6 +44,21 @@ If the schema does not already exist and the `db-init-script` is not in the defa
 
 Where `/tmp/schema.sql` contains: `CREATE SCHEMA IF NOT EXISTS my_db;`
 
+Full list of configuration options:
+|Property|Required|Description|
+|----|----|----|
+|`housekeeping.expired-path-duration`|No|Time To Live (TTL) of legacy replica paths in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) format: only days, hours, minutes and seconds can be specified in the expression.|
+|`housekeeping.schema-name`|Yes|Database schema name to use. Tables will be created or used (if already existing) using this schema. Default: 'housekeeping'|
+|`housekeeping.db-init-script`|No|Database init script to use. Default: 'classpath:/schema.sql'
+|`housekeeping.data-source.driver-class-name`|No|Java classname of the database JDBC driver.|
+|`housekeeping.data-source.url`|No|JDBC connection URL.|
+|`housekeeping.h2.database`|No|If the `housekeeping.data-source.url` is not overridden the defult H2 database is configured. The folder where H2 will write the database files is configured with this property. Defaults to `${instance.home}/data/${instance.name}/housekeeping`. Where `instance.home` and `instance.name` can be configured separately for more fine grained control.|
```

Comment:
How about this?

> 
|`housekeeping.h2.database`|No|If the `housekeeping.data-source.url` is not overridden then the default H2 database can be configured using this property which also controls where H2 will write its database files. Defaults to `${instance.home}/data/${instance.name}/housekeeping` (where `instance.home` and `instance.name` can be configured separately for more fine grained control).|

Reply:
^ Fixes a typo and hopefully makes it a bit clearer (if I'm understanding it correctly)

Reply:
yes better, thanks